Description: Rename the config file to be more specific
 A config file named "config" might be fine if it's inside the software's own
 install folder, like Windows often does. But `/etc` is the system config dir
 and will include all kinds of things. The name needs to be more descriptive.
 .
 unifi-monitor (0.0~git20250105.9d5b8a64-1) UNRELEASED; urgency=low
 .
   * Experimental package manifest
Author: Robert Garrett <robertgarrett404@gmail.com>

---

Bug: https://github.com/authrequest/Unifi-Monitor/issues
Forwarded: no
Last-Update: 2025-01-06

--- unifi-monitor-0.0~git20250105.9d5b8a64.orig/all_products.go
+++ unifi-monitor-0.0~git20250105.9d5b8a64/all_products.go
@@ -343,7 +343,7 @@ func readEnv(key, defaultValue string) s
 		return value
 	}
 
-	configFile := "/etc/config.yml"
+	configFile := "/etc/unifi-monitor.yml"
 	if _, err := os.Stat(configFile); err == nil {
 		data, err := ioutil.ReadFile(configFile)
 		if err == nil {
